README.md

```diff
@@ -47,10 +47,14 @@ To install, run:
 curl -fsSL https://coder.com/install.sh | sh
 ```
 
-Once installed, you can run a temporary deployment in dev mode (all data is in-memory and destroyed on exit):
+Once installed, you can start a production deployment with a single command:
```

Comment:
Sorry, but turning the DB off if the single instance of the app goes down is not what a lot of people would consider "production"

Reply:
Why not?

Reply:
We do essentially this same thing right now on our dogfood deployment, which I'd consider a production deployment of Coder.

Reply:
I agree with @f0ssel here -- I'd be more comfortable with a "traditionally-managed" database if I were setting up a prod deployment than just using the embedded one. Doesn't detract from the value of being able to instantly kick the tyres with a real database!

Reply:
Why though? I believe users should start real, production deployments with this setup. There isn't a technical reason why it's bad practice. Users can just backup the `~/.config/coder/postgres` directory on a CRON.

Reply:
> Why though? 

I don't have a good answer for that 😅 apart from wanting my system to be in control of the DB, and nothing else.

> Users can just backup the ~/.config/coder/postgres directory on a CRON.

![image](https://user-images.githubusercontent.com/4949514/173910725-ea300b1d-ecb7-4caa-a652-08db1151e6fe.png)

Reply:
> I don't have a good answer for that sweat_smile apart from wanting my system to be in control of the DB, and nothing else.
I will buy this answer because there is some level of that. We don't prevent you from doing that though, so this solution seems reasonable!

Yes, but you could still do:
```
sudo service coder stop
# Backup
sudo service coder start
```

Reply:
>  We don't prevent you from doing that though, so this solution seems reasonable!

Oh it's absolutely a reasonable solution -- just "production-ready" is a very subjective term :-)

Reply:
Fair fair!

Reply:
Coupling the lifecycle of the database and running the DB on the same host are two pretty big red flags for any engineers that have to actually manage and troubleshoot the app in production.

Reply:
What are the specific red flags? You can still run the database independently.

Reply:
I'm saying that if a customer wanting to run 1000 devs and it be production ready I'd probably advise them:
- Run the DB on a dedicated host
- Run regular database backups and have a restore process
- Run multiple instances of the app server, optionally on multiple hosts behind a LB, optionally across different AZs. 
- Have a way to run new webserver versions without causing downtime

Right now this command not only does none of those things, but always won't even allow you to run multiple instances of the web app since it will conflict on the DB setup.

Reply:
And similar to why people recommend not putting DB healthchecks in the webserver healthchecks on kubernetes - you don't want your DB taking your webapp down and vice versa, you want their lifecycles independent for obvious reasons.